test: `TestResourceVariable_namespaceChange` failing

Test failure:
```
=== RUN   TestResourceVariable_namespaceChange
    testing_new.go:85: Error running post-test destroy, there may be dangling resources: exit status 1
        
        Error: error deleting namespace "var-test-namespace": Unexpected response code: 500 (namespace "var-test-namespace" contains at least one variable. All variables in namespace must be deleted before it can be deleted)
```

--- a/nomad/resource_variable_test.go
+++ b/nomad/resource_variable_test.go
@@ -98,7 +98,7 @@ resource nomad_namespace "nomad_var_test" {
   name = "%s"
 }
 %s
-`, namespace, testResourceVariable_initialConfig(namespace, path))
+`, namespace, testResourceVariable_initialConfig("${nomad_namespace.nomad_var_test.name}", path))
 }
 
 func testResourceVariable_initialCheck(namespace, path string) resource.TestCheckFunc {

